src/Group/Command/CreateGroupHandler.php

```diff
@@ -47,8 +45,8 @@ public function handle(CreateGroup $command)
         $actor = $command->actor;
         $data = $command->data;
 
-        $this->assertRegistered($actor);
-        $this->assertCan($actor, 'createGroup');
+        $actor->assertRegistered();
+        $actor->assertCan('createGroup');
```

Comment:
it's odd to have two checks. there should be single policy for this.

Reply:
It's intentional, in order to provide different status codes depending on the exact situation. See commit 3b5691ee2807bcc917b89f86ba99f4a4725095dc.